pkg/bootkube/create.go

```diff
@@ -80,15 +80,19 @@ func createAssets(manifestDir string) error {
 		return err
 	}
 
-	includeExtendedAPIs := false
-	mapper, typer := f.Object(includeExtendedAPIs)
+	// aaron: is this ok that we can't specify the extend api option anymore?
```

Comment:
I took a look at https://github.com/kubernetes/kubernetes/blob/v1.5.0-beta.3/pkg/kubectl/cmd/create.go#L100

As far as the section I was changing it looks pretty similar but notably the kubectl code calls `f.UnstructuredObject()`. Also the upstream visit function looks quite a bit different, but i'm not very familiar with this code.